### PR TITLE
Migration of Chromium Chronicle articles & Add float-(left|right) 

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Migrates the [Chromium Chronicle posts](https://developers.google.com/web/updates/tags/chromium-chronicle) from d.g.c

Need the compare widget implemented (#213), and the image float left/right style (#212)

### CODE REVIEW

File: site/en/blog/chromium-chronicle-1/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  The Chrome team is proud to introduce the Chromium Chronicle, a monthly
+  series geared specifically to Chromium developers - the developers who
+  build the browser. This month, we take a look at task scheduling best
+  practices.
+layout: 'layouts/blog-post.njk'
+date: 2019-04-16
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+---
```
Comment (petele) [inline site/en/blog/chromium-chronicle-1/index.md:13]: Want to be able to add tags to tag all of these as `chromium-chronicle`

File: site/_data/i18n/tags.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ accessibility:
 amp:
   en: AMP
 
+chromium-chronicle:
```
[the remark this answers is not in the document]
Comment (petele) [reply site/_data/i18n/tags.yml:14]: Happy to change, but looking at the file, I see all other instances use hyphen, nothing is using underscore.

File: site/_data/supportedTags.json [hunk 3]
```diff
@@ -11,6 +11,9 @@
   "amp": {
     "title": "i18n.tags.amp"
   },
+  "chromium-chronicle": {
```
[the remark this answers is not in the document]
Comment (petele) [reply site/_data/supportedTags.json:14]: See note above.

File: site/_data/i18n/tags.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (robdodson) [reply site/_data/i18n/tags.yml:14]: oh you're right, my mistake. Ignore me!

**Comment by petele** [comment]
@robdodson - this is now ready to go! Thanks for all your help on it.

**Pull request merged**